_posts/2024-05-05-breaking-cultural-barriers.html

```diff
@@ -0,0 +1,36 @@
+    <img class="img-fluid" src="/assets/images/blog/breaking-barriers/quote2.png" alt="quote2">
+    <p>Before long, I found myself diving into a Full Stack developer bootcamp. It was a game-changer. Coding made me feel alive again. It wasn't just about learning a new skill—it was about reclaiming a part of myself that I had lost. </p>
+    <p>The more I coded, the more my confidence grew. Coding empowered me in ways I never imagined. It reminded me that it's never too late to pursue our passions and shape our own futures.</p>
+    <p>After finishing my bootcamp, I eagerly started my job search, only to face disappointment. But instead of feeling defeated, I decided to change my perspective.</p>
+    <p>With over 15 years of no corporate experience, I realised that I could aim for roles where it allows me to learn tech in general and have some kind code aspects as well.</p>
+    <p>This shift in thinking opened new opportunities for me. Instead of focusing solely on software developer roles, I started looking for positions where I could gain exposure to coding and other tech areas.</p>
+    <p>After an extensive search for career opportunities, an agency sent me a link to a role at Network Rail that immediately caught my attention. Although the role wasn’t a software developer, it presented a unique blend of responsibilities that excited me, and it mentioned managing intranet sites which I felt would have some kind of development aspect in it. </p>
+    <img class="img-fluid" src="/assets/images/blog/breaking-barriers/2024-05-05-main-image.jpg" alt="breaking-barriers-photo">
+    <p>While my current position isn't solely focused on software development, I still have the opportunity to work with TypeScript, React, JavaScript, and JSON using SharePoint framework. Recently, I've been entrusted with the exciting project of re-developing our department’s entire SharePoint Intranet site.</p>
+    <p>From creating wireframes and prototypes to overseeing the entire project, I'm responsible for every aspect. It's an incredible opportunity, and I'm thoroughly enjoying every moment of it.</p>
+    <p>As an older woman diving into the world of tech/software development, I often feel the weight of time pressing down on me as I am getting older. It's easy to get caught up in the rush to achieve more, but my mentor's advice to take things "one step at a time" resonates deeply. I've come to realize that the journey is just as important as the destination. There are days when the energy levels aren't quite what they were in my 20s or 30s, especially with the challenges of perimenopause. Despite this, I strive to appreciate the journey and find joy in learning.</p>
+    <img class="img-fluid" src="/assets/images/blog/breaking-barriers/quote3.jpg" alt="quote3">
```

Comment:
In my opinion 3 quotes with the same banner image is too much, what about just add the quote text and footer? Without the laptops. What do you think?

Reply:
Very good point! 
I was thinking about it and from the conversation with Lily in the channel found out that there is a blockquote component existing. I added it instead of this image. 
Styling will be a little bit tweaked later, when figma is ready :) What do you think?
<img width="799" alt="Screenshot 2024-05-07 at 16 33 31" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/65826199/5dabf0d5-962c-4488-9c86-fb777c88727a">
<img width="721" alt="Screenshot 2024-05-07 at 16 39 54" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/65826199/51822290-12ab-40cb-85b0-da4804b1141d">